src/com/owncloud/android/ui/helpers/FilesUploadHelper.java

```diff
@@ -0,0 +1,224 @@
+/**
+ *   ownCloud Android client application
+ *
+ *   @author Christian Schabesberger
```

Comment:
I would include here Shashvat Kedia as second @author since he is the author of great part of this code refactored from UploadFilesActivity